en/05_Contributing/17_Managing_Security_Issues.md

```diff
@@ -0,0 +1,113 @@
+* Move the issue to "Done" in the [project board](https://github.com/silverstripe-security/security-issues/projects/1)
+
+### Detailed CVE and CVSS Guidance
+
+* In the GitHub _draft security advisory_, we follow certain conventions on fields:
+  * **Affected product** should have one entry for each composer package affected by the vulnerability. If the patch is backported to older release lines, there should be entries for those older releases.
+    * **Ecosystem** should always be `composer`.
+    * **Package name** sholud match the package name in Packagist.
+    * **Affected versions** should be a string reprenting a range of version affected by the vulnerability.
+    * **Patched versions** should be a list version where the vulnerability was/will be patched.
+  * **Severity** should be set to *Assess severity using CVSS* and the CVSS option should be specified in the calculator.
+  * **Common weakness enumerator (CWE)** should be left blank if no suitable option can be found in the list.
+  * **CVE identifier** should be set to "Request CWE ID later".
+  * **Title** and **Description** should be descriptive enough that people can evaluate the risk without being so specific that would-be-hackers can start exploiting them right away with no efforts. 
+
+These settings can be changed later, so you can iterate over them.
+
+## Special circumstances
+
+### An issue has been reported publicly on GitHub
+
+Occasionally, some developers will open a public issue on GitHub reporting a vulnerability, either because they are unaware of the proper process or because they did not immediately grasp the security implication of the bug being reported.
+
+In those situations, the security team will assess how likely it was that the vulnerability was widely noticed based on how long the issue was opened on GitHub and how explicit the description was. If the risk of the vulnerability having been noticed is small, an owner on the Silverstripe GitHub organisation will delete the public facing issue and create a private one instead.
+
+### Handling issues on Huntr
+
+[Huntr.dev](https://huntr.dev/) is an open source vulnerability bounty platform. The security team occasionally receives vulnerability reports from Huntr. Reports from Huntr go through more or less the process as the regular reports. The security team should take the following points into consideration when working on a Huntr issue:
+
+- While you can reply via email and the response will be recorded in Huntr, who sent the response will not. To keep the discussion flowing more naturally, it is preferable to login into Huntr with your GitHub account and respond via the Huntr interface.
+- People reporting vulnerabilities via Huntr are not always very familiar with Silverstripe CMS. Don't assume that they have a lot of context on what Silverstripe CMS is used for.
```

Comment:
```suggestion
- People reporting vulnerabilities via Huntr are not always familiar with Silverstripe CMS. Don't assume that they have much context on what Silverstripe CMS is used for.
```

Reply:
Please remove "very" or explain why you don't want to.

Reply:
I've removed the "very". 

Like the rest of this section, this point is not meant to be prescriptive. The bugs we get reported via this channel come from people that might not fully understand the impact of what they are reporting and off the mitigating circumstances.